ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/LfValueTranslation.scala

```diff
@@ -171,10 +176,34 @@ final class LfValueTranslation(val cache: LfValueTranslation.Cache) {
 
 object LfValueTranslation {
 
-  type Cache = caching.Cache[Cache.Key, Cache.Value]
+  final case class Cache(events: EventCache, contracts: ContractCache)
```

Comment:
This singular name is a bit confusing here. Maybe call it `Caches` or `CachedEntities` or similar to be explicit that it is holding more than one cache?